packages/opentelemetry-cloud-trace-exporter/src/trace.ts

```diff
@@ -80,7 +80,7 @@ export class TraceExporter implements SpanExporter {
     resultCallback(result);
   }
 
-  shutdown(): void {}
+  async shutdown(): Promise<void> {}
```

Comment:
I think we should add some logic here to set the shutdown flag to true when this method is called and in export return immediately when the flag is set to true. Currently, shutdown operation looks more like a no-op. WDYT? Perhaps, we can do this in separate PR.

Reply:
Makes sense, I haven't really look into it yet -- tracking in https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/issues/190